diff --git a/src/terminal/Screen.zig b/src/terminal/Screen.zig
index 0772bfa7..28a5c9ac 100644
--- a/src/terminal/Screen.zig
+++ b/src/terminal/Screen.zig
@@ -2537,6 +2537,10 @@ pub fn selectWord(self: *Screen, pin: Pin) ?Selection {
         '<',
         '>',
         '$',
+        '⏎',
+        '«',
+        '»',
+        '=',
     };
 
     // If our cell is empty we can't select a word, because we can't select
