diff --git a/layout/centerfair.lua b/layout/centerfair.lua
index 5022726..5c63cfa 100644
--- a/layout/centerfair.lua
+++ b/layout/centerfair.lua
@@ -70,6 +70,7 @@ function centerfair.arrange(p)
     if #cls < num_x
     then
         -- Less clients than the number of columns, let's center it!
+        if #cls == 1 then width = width * 1.5 end
         local offset_x = wa.x + (wa.width - #cls*width - (#cls - 1)*useless_gap) / 2
         local g = {}
         g.y = wa.y + useless_gap
